apps/condo/pages/ticket/index.tsx

```diff
@@ -51,6 +60,7 @@ const ROW_GUTTER: [Gutter, Gutter] = [0, 40]
 const TAP_BAR_ROW_GUTTER: [Gutter, Gutter] = [0, 20]
 const CHECKBOX_STYLE: CSSProperties = { paddingLeft: '0px', fontSize: fontSizes.content }
 const TOP_BAR_FIRST_COLUMN_GUTTER: [Gutter, Gutter] = [40, 20]
+const ROW_GUTTER_10_0: [Gutter, Gutter] = [10, 0]
```

Comment:
Useless const name, make it more obvious by specifying field of usage

Reply:
Why bind it to a specific element? Because of this, it will not be possible to reuse the variable.

Reply:
Many components have constants for gutters. But they are repeated. Wouldn't it be better to define them separately and reuse them?
Why create new repeated constants?

Reply:
How should I understand, which constant should I reuse? `GUTTER_10_0` tells nothing about it. If it was something like `PAGE_SPACING_MEDIUM` it will be more understandable

Reply:
What does "MEDIUM" mean? How much is this? 10, 20, 30, 40? Unclear